core/src/main/java/org/springframework/security/converter/RsaKeyConverters.java

```diff
@@ -50,6 +56,12 @@
 
 	private static final String X509_PEM_FOOTER = DASHES + "END PUBLIC KEY" + DASHES;
 
+	private static final String X509_CERT_HEADER = DASHES + "BEGIN CERTIFICATE" + DASHES;
+
+	private static final String X509_CERT_FOOTER = DASHES + "END CERTIFICATE" + DASHES;
+
+	public static final String X509_CERTIFICATE = "X.509";
```

Comment:
I think this string should be inlined so that it remains consistent with existing code.